src/Core/Constants.cs

```diff
@@ -41,6 +41,7 @@ public static class Constants
         /// which is used to handle Apple Watch state logic
         /// </summary>
         public const string LastUserShouldConnectToWatchKey = "lastUserShouldConnectToWatch";
+        public const string AvatarColorKey = "avatarColor";
```

Comment:
Since this is a user-specific pref, it should be added below (following the `_{userId}` pattern)